googler

```diff
@@ -3338,8 +3350,10 @@ def parse_args(args=None, namespace=None):
            help='disable TCP optimizations and forced TLS 1.2')
     addarg('--json', action='store_true',
            help='output in JSON format; implies --noprompt')
-    addarg('--url-handler', metavar='UTIL',
+    addarg('--url-handler', metavar='URL_HANDLER',
```

Comment:
What's the problem with `UTIl`. This has to be a utility.

Reply:
I didn't realize exactly what you meant by a utility.  I changed it back now.